profiles/thurloe/terraform/thurloe/dns.tf

```diff
@@ -0,0 +1,45 @@
+provider "google" {
+  alias = "dns"
+  project     = "${var.dns_project}"
+  region      = "${var.dns_region}"
+  credentials = "${file("dns_sa.json")}"
+}
+
+# Cloud SQL dns
+resource "google_dns_record_set" "mysql-instance-new" {
```

Comment:
Not sure I understand why you have the -new and below the -old resources?  Seems like if you are making a structural change on how DNS is done - just do it.  These environments are "ephemeral", destroy/build cycle seems reasonable.

Reply:
The `-old` ones should just be deleted, I'm going to fix that. The problem with `-new` is that if I just change from a local-to-the-project DNS to a dsde-perf DNS I have to change the provider. If I change the provider but leave the resource name the same, it will try to use the new provider to delete the old resource, which results in a 403. So it's a migration thing--we can remove the `-new` from the name in a later PR.